packages/affiliates/apps/DevMiningRewards.js

```diff
@@ -81,16 +85,13 @@ async function App(config, env) {
   });
 
   return {
-    config,
+    ...params,
     // result will contain deployer rewards as well as per emp rewards
-    ...result
+    result
   };
-}
-
-const config = Config();
+};
 
-App(config, process.env)
-  .then(x => console.log(JSON.stringify(x, null, 2)))
+makeUnixPipe(App(process.env))
```

Comment:
again converted from a config file to a json blob piped through standard in

Reply:
this is cool.